### PR TITLE
Show new event button in Agenda view when it's empty (for single and two columns layout)

### LINKED ISSUE #7139: New event button at the Agenda view when it's empty
Add a new event button in the Agenda view when it's empty, bellow the `<No entries>` message

### Test notes
- [ ] Check if New Event button show up at the agenda view when it's empty, but not in Desktop (3 columns view) since it's already visible in the sidebar
- [ ] Creating an event through this button should work as expected

### PR DESCRIPTION
**Pull request by andrehgdias**
Closes #7139 
image

### CODE REVIEW

File: src/gui/base/buttons/BaseButton.ts [hunk 1]
```diff
@@ -61,3 +64,20 @@ export class BaseButton implements ClassComponent<BaseButtonAttrs> {
 function booleanToAttributeValue(value: boolean | null | undefined): string | null {
 	return value != null ? String(value) : null
 }
+
+export function renderNewEventButton(buttonAttrs: { label: TranslationKey; click: ClickHandler; style?: Record<string, any> }): Children {
```
Comment (murilopereirame) [inline src/gui/base/buttons/BaseButton.ts:68]: The name should be more generic in this case since this button is also used to create a new email.

Comment (murilopereirame) [inline src/gui/base/buttons/BaseButton.ts:68]: This isn't a base button, it's worthwhile to have it's own file

File: src/calendar/view/CalendarAgendaView.ts [hunk 2]
```diff
@@ -184,6 +186,13 @@ export class CalendarAgendaView implements Component<CalendarAgendaViewAttrs> {
 				icon: BootIcons.Calendar,
 				message: "noEntries_msg",
 				color: theme.list_message_bg,
+				bottomContent: renderNewEventButton({
+					label: "newEvent_action",
+					click: () => attrs.onCreateEvent?.(),
+					style: {
+						"margin-top": ".6rem",
```
Comment (murilopereirame) [inline src/calendar/view/CalendarAgendaView.ts:193]: We have some utility classes for sizing and margins that we should use to keep the spacing standardized, take a look at [main-styles.ts#L266](https://github.com/tutao/tutanota/blob/971c09f66b0055e5b1af7684db53b6265c4c5ab0/src/gui/main-styles.ts#L266)

Comment (murilopereirame) [inline src/calendar/view/CalendarAgendaView.ts:193]: We have some utility classes for sizing and margins that we should use to keep the spacing standardized, take a look at [main-styles.ts#L266](https://github.com/tutao/tutanota/blob/971c09f66b0055e5b1af7684db53b6265c4c5ab0/src/gui/main-styles.ts#L266)

**Review by murilopereirame — CHANGES_REQUESTED** [review]

**Review by murilopereirame — APPROVED** [review]

**Review by wrdhub — APPROVED** [review]

**Review by rih-tutao — APPROVED** [review]

**Pull request closed without merging**